### ISSUE TITLE
3.0.1 GUI Local path settings have no effect on the server

### ISSUE THREAD

**Issue by nuqleo** [GNS3/gns3-gui#3687]
**Describe the bug**
After changing Local path settings gns3-server does not use new paths.

**GNS3 version and operating system (please complete the following information):**
 - OS: Linux
 - GNS3 version 3.0.1

**To Reproduce**
1. Change  'Local path settings'
2. GUI uses new paths but gns3-server don't.

**Additional context**
2.2.52 GUI creates gns3_server.conf with *_path aettings but 3.0.1 does not create this file.
If I create gns3_server.conf with new path then gns3-server also uses this paths.

**Comment by smart2128** [comment]
https://github.com/GNS3/gns3-gui/issues/3677

**Comment by grossmj** [comment]
Indeed, this doesn't work anymore. I wanted to add an API endpoint to configure the server settings but I haven't got a chance to implement it.
